cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -275,7 +290,25 @@ public Secret generateNodesSecret() {
                 .withResources(resources())
                 .build();
 
+        ResourceRequirements resources = new ResourceRequirementsBuilder()
+                .addToRequests("cpu", new Quantity("100m"))
+                .addToRequests("memory", new Quantity("128Mi"))
+                .addToLimits("cpu", new Quantity("1"))
+                .addToLimits("memory", new Quantity("128Mi"))
+                .build();
```

Comment:
How did you determine these? Do you know they're sufficient no matter what the load, or that the container is not overresourced? Do they need to be configurable?

Reply:
We didn't make them configurable for the Kafka init container as well. I did some simple tries but never went under 128MB (Maybe it could work with 64MB as well). For sure I didn't run a load test for that.